packages/dara-core/pyproject.toml

```diff
@@ -50,7 +51,7 @@ openpyxl = "*"
 packaging = "^23.1"
 pandas = ">=1.1.0, <3.0.0"
 prometheus-client = "^0.14.1"
-pyarrow = "*"
+pyarrow = "<17.0.0"
```

Comment:
What's the reason to restrict pyarrow here?

Reply:
Was just testing, the version had just been released and was not being picked up yet so was just checking the issue was not elsewhere. Seems to be picking it up now though so have removed it